tests/e2e/normalize.go

```diff
@@ -106,6 +106,20 @@ func normalizeKRMObject(u *unstructured.Unstructured, project testgcp.GCPProject
 	// Specific to Compute SSL Certs
 	visitor.replacePaths[".status.observedState.certificateId"] = "1.719337333063698e+18"
 
+	// Specific to MonitoringDashboard
+	visitor.stringTransforms = append(visitor.stringTransforms, func(path string, s string) string {
+		if strings.HasSuffix(path, ".alertChart.alertPolicyRef.external") {
+			tokens := strings.Split(s, "/")
+			if len(tokens) > 2 {
+				switch tokens[len(tokens)-2] {
```

Comment:
I guess the second to last token is always `alertPolicies`, right?

Reply:
If it's an alertPolicy, I believe so.    It's normalization for tests, so it's not the end of the world if we're wrong.

(And similar to what you suggested the other day, I think we may be able to factor out some common logic here in future.  The other option though is to build the replacements by looking at the http log, and share them with object normalization, rather than having to figure them out each time.  TBD, I guess!)